### ISSUE TITLE
Sign Up doesn't work

### ISSUE THREAD

**Issue by karbiv** [nim-lang/nimforum#373]
"Couldn't send email".

**Comment by dinesh1011** [comment]
I am facing same issue in signing up to the Nim forum. Initially the error "couldn't send email" was thrown and latter after few tries facing "Too many messages have been sent to this email address recently.".
anyways, Have received zero emails for signup and not able to complete signup.

**Comment by dinesh1011** [comment]
Is there a way to add nicknames to the forum?
Anyhow nick name start with "DK" you can use.
Ex. DKnim, DK001, DK....,

**Comment by PMunch** [comment]
I meant nickname is in username, sorry for the confusion. We're trying to fix this issue, its something going on with the e-mail sending API we're using. For now we can manually register you though if you reach out to the moderators in private chats on Discord/Matrix/IRC.

**Comment by karbiv** [comment]
> We're trying to fix this issue, its something going on with the e-mail sending API we're using. For now we can manually register you though if you reach out to the moderators in private chats on Discord/Matrix/IRC.

No hurry. Github is enough so far.

**Comment by agrin** [comment]
Hello!  
I’m having the same issue. I’m trying to register on the forum, but it keeps giving me the same error.

**Comment by PMunch** [comment]
> We're trying to fix this issue, its something going on with the e-mail sending API we're using. For now we can manually register you though if you reach out to the moderators in private chats on Discord/Matrix/IRC.

@agrin

**Comment by agrin** [comment]
Thanks for replying, I can't find any moderator on Discord, but no problem, I'm waiting for the mailing API to be solved.
If it is possible that you let us know here, it will be appreciated.

**Comment by PMunch** [comment]
Can't find any? You could just PM me on Discord, I'm a moderator, same nicname there as here.

I'll make sure to update this post once the issue is fixed

**Comment by PMunch** [comment]
Yes, we are aware. Please contact a moderator in any of our other channels to create your user. We're working on fixing the issue

**Comment by PMunch** [comment]
Good idea, I've been meaning to add a note on the log-in page, but that's more work than a stickied topic. If you PM me your details on any of our platforms I can set you up with an account.

**Comment by PMunch** [comment]
Yes, the message came through. Just created your account and sent you an e-mail with the password.

**Comment by PMunch** [comment]
This has been resolved now. It should be possible to sign up once again using the normal e-mail route. Thanks for your patience!
